client/common/src/types.rs

```diff
@@ -62,96 +59,15 @@ impl AsRef<[u8; 32]> for FileKey {
     }
 }
 
-#[derive(Encode, Decode, Clone, Copy, Debug, PartialEq, Eq, Default, Serialize, Deserialize)]
-pub struct Fingerprint(Hash);
-
-impl Fingerprint {
-    /// Returns the hash of the fingerprint.
-    pub fn hash(&self) -> Hash {
-        self.0
-    }
-}
-
-impl From<Hash> for Fingerprint {
-    fn from(hash: Hash) -> Self {
-        Self(hash)
-    }
-}
+pub type KeyVerifier = <Runtime as pallet_proofs_dealer::Config>::KeyVerifier;
+pub type FileKeyProof = <KeyVerifier as CommitmentVerifier>::Proof;
 
-impl Into<Hash> for Fingerprint {
-    fn into(self) -> Hash {
-        self.0
-    }
-}
+pub const H_LENGTH: usize = BlakeTwo256::LENGTH;
```

Comment:
How about defining this in `runtime/src/configs/mod.rs` and importing from there to here? In case we change hasher for instance (will hardly be the case, but since we're at it...)